app/helpers/ops_helper.rb

```diff
@@ -37,6 +37,20 @@ def advanced_tab_warning
     end
   end
 
+  def advanced_tab_info
+    parent =
+      if selected?(x_node, "z")
+        "Region"
+      elsif selected?(x_node, "svr")
+        "Zone"
+      else
+        "Server"
+      end
+
+    _('If you would like to reset a value or section in order to inherit from the parent %{parent}, or to delete' \
+      ' a value that is not present in the parent %{parent}, set the value to <<reset>>.')
```

Comment:
This should be:

```ruby
_("If you would like to ... %{parent}, or to delete ... %{parent}") % {:parent => parent}
```

right? Or at least that's what I gather was the intention here.

Reply:
ohhh...it uses `%` operator.  I was trying `_("If you .. %{parent} ...", :parent => parent)`, which seemed more intuitive to me.